src/init.c

```diff
@@ -76,17 +76,16 @@ void jl_init_stack_limits(int ismaster, void **stack_lo, void **stack_hi)
 #if defined(_COMPILER_GCC_) && __GNUC__ >= 12
 #pragma GCC diagnostic ignored "-Wdangling-pointer"
 #endif
-        *stack_hi = (void*)&stacksize;
```

Comment:
why are we using `&stacksize` rather than the value of it as `stackaddr + stacksize`

Reply:
I don't know, but the former is slightly smaller of course. I guess it may prevent us from smashing any extra information the system may have put at the stack base.

Reply:
we should never smash this; we just use it to decide if it hit a StackOverflow